core/data_operations/round_data.py

```diff
@@ -0,0 +1,58 @@
+"""Dummy data to use with testing
+"""
+from core.dummy_dao import RoundDAO
+
+
+ROUND_DATA = [
+    {
+        "round_id": "spring",
+        "round_title": "Spring",
+        "fund_id": "funding-service-design",
```

Comment:
Assuming the round data and fund data will be linked at some future point via a primary-foreign key relationship in db. Should we decide on either 'fund_id' or 'fund_name' across both the fund and round store as per standard SQL best practices.